pkgs/applications/system/monitor/default.nix

```diff
@@ -90,7 +90,7 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/stsdc/monitor";
     maintainers = with maintainers; [ xiorcale ] ++ teams.pantheon.members;
     platforms = platforms.linux;
-    license = licenses.gpl3;
+    license = licenses.gpl3Plus;
```

Comment:
where did you get the "plus" from? i only see GPLv3

Reply:
https://github.com/stsdc/monitor/blob/0.13.0/data/com.github.stsdc.monitor.appdata.xml.in#L6